catalog/view/theme/default/template/payment/razorpay.tpl

```diff
@@ -15,6 +15,7 @@
     notes: {
       opencart_order_id: "<?php echo $merchant_order_id; ?>"
     },
+    callback_url: "<?php echo $return_url; ?>",
```

Comment:
@pronav want to confirm the priority order for `handler` and `callback_url` being present.